telemetry/metrics.yaml

```diff
@@ -138,9 +89,23 @@ ui:
           with by the user, if any.
           The Mastodon API calls statuses "posts", but they are one and the same.
         type: string
-      corpus_recommendation_id: &corpus_recommendation_id
+      mastodon_account_id: &mastodon_account_id
+        description: >
+          The Mastodon account ID of the UI element that was seen/interacted
+          with by the user, if any.
+          For example, the account ID of the user who created a post that was
+          seen/interacted with. Or, the account ID of the user that was followed.
+        type: string
+      mastodon_account_handle: &mastodon_account_handle
+        description: >
+          The Mastodon account handle of the UI element that was seen/interacted
+          with by the user, if any.
+          For example, the account handle of the user who created a post that was
```

Comment:
I'm getting confused because there's parts in the file that say
> The user's full account handle, with domain.

Does saying only "account handle" refer to, for example, `account_name` in `account_name@mozilla.social`?

Reply:
Any time it's talking about the mastodon account handle, we should be passing the full account link (for example: `account_name@mozilla.social`). That's part of what we are optimizing in this PR - we were going to split it out into two identifiers `mastodon_instance_domain` and `mastodon_account_handle`, but Kirill realized that if we just always pass the full account link for the `mastodon_account_handle`, we will have all the info we need.

Here's an example of what we were passing before:
```
      "identifiers.mastodon_instance_domain": "mozilla.social",
      "identifiers.mastodon_account_handle": "alytest01"
```

And what we are sending now:
```
      "identifiers.mastodon_account_handle": "alytest01@mozilla.social"
```